show/main.py

```diff
@@ -15,9 +15,15 @@
 import sonic_platform
 from swsssdk import ConfigDBConnector
 from swsssdk import SonicV2Connector
+from portconfig import get_port_config
```

Comment:
Why add this import if you are not using it?

Reply:
removed in new commit

Reply:
? This was not removed.

Reply:
Addressed in new commit!